pom.xml

```diff
@@ -1415,14 +1415,27 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
-                        <version>3.8.0</version>
                         <!--suppress MavenModelInspection -->
                         <configuration combine.children="merge">
-                            <fork>true</fork>
```

Comment:
was it the reason why it didn't work for me?

Reply:
Yep: https://issues.apache.org/jira/browse/MCOMPILER-425